packages/services/f-feature-management/src/vue/services/features.service.js

```diff
@@ -77,20 +77,24 @@ export default class FeaturesService {
         }
     ) {
         this.store = store;
+        this.logger = createLoggerWrapper(store, logger);
 
         store.registerModule(namespace, featuresModule, { preserveState: !!this.serviceStore });
 
-        const loggerWrapper = createLoggerWrapper(store, logger);
-        const settings = createSettings(this.store, this.serviceStore, cookies, analytics, loggerWrapper);
+        const settings = createSettings(this.store, this.serviceStore, cookies, analytics, this.logger);
         this.featureManagement = createFeatureManagementInstance(settings, httpClient);
     }
 
     /**
      * Loads the latest features configuration from the CDN
      */
     async update () {
-        const newConfigJson = await this.featureManagement.loadFromCdn();
-        this.store.dispatch(storeAction, newConfigJson);
+        try {
+            const newConfigJson = await this.featureManagement.loadFromCdn();
+            this.store.dispatch(storeAction, newConfigJson);
```

Comment:
As [`dispatch()` returns a `Promise`](https://vuex.vuejs.org/api/#dispatch), it might be worth `await`ing this if we want to catch potential problems with the action as well.